RoddyCore/src/de/dkfz/roddy/tools/RuntimeTools.groovy

```diff
@@ -37,4 +48,20 @@ final class RuntimeTools {
         return new File(System.getProperty("user.dir"), "dist/bin/current")
     }
 
+    static File getGroovyLibrary() {
+        // Try to get Groovy from the environment. This is needed for groovyserv.
+        // If this is not working get it from the classpath.
+        logger.rare(([""] + System.getenv().collect { String k, String v -> "${k}=${v}" }.join("\n") + [""]).flatten().join("\n"))
```

Comment:
Will this work with the Gradle build, when no groovy library is e.g. in the lib folders?

Reply:
Tried it once and worked. I think I will try this also  out with my AQCWF integration tests.